ledger/src/blockstore_db.rs

```diff
@@ -103,6 +102,21 @@ const OPTIMISTIC_SLOTS_CF: &str = "optimistic_slots";
 /// Column family for merkle roots
 const MERKLE_ROOT_META_CF: &str = "merkle_root_meta";
 
+macro_rules! concat_key_bytes {
+    ($key:ident, $($range:expr => $bytes:expr),* $(,)?) => {{
+        let mut key = [0u8; std::mem::size_of::<Self::$key>()];
+        debug_assert_eq!(0 $(+$bytes.len())*, key.len());
+        $(key[$range].copy_from_slice($bytes);)*
+        key
+    }};
+}
+
+macro_rules! convert_key_bytes {
```

Comment:
nit: The usage of the macro makes it pretty clear what purpose the macro serves, but it is a little less clear just looking at the macro in isolation.

Thoughts on a one-line comment and/or slightly more descriptive name, maybe something like `convert_column_key_bytes_to_index`. Could possibly rename the other one to something like `convert_column_index_to_key_bytes` so the names are "symmetric"

Reply:
done!